ai_eval/templates/chatbox.html

```diff
@@ -26,10 +26,13 @@
         </div>
         <div class="submit-row">
             {% if self.allow_reset %}
-                <span id="reset-button" class="disabled-btn">Reset chat</span>
+                <span class="chat-button" id="reset-button">Reset chat</span>
             {% endif %}
-            <textarea class="user-input" rows="1" placeholder="Type your answer here" maxlength="1000"></textarea>
-            <span id="submit-button">Submit <i class="fa fa-paper-plane"></i></span>
+            {% if has_finish_button %}
+                <span class="chat-button" id="finish-button">Submit for evaluation</span>
```

Comment:
@ArturGaspar It might be better to disable this button until there's a user response. What do you think?